swarm/pss/api.go

```diff
@@ -157,14 +157,23 @@ func (pssapi *API) StringToTopic(topicstring string) (Topic, error) {
 }
 
 func (pssapi *API) SendAsym(pubkeyhex string, topic Topic, msg hexutil.Bytes) error {
+	if !checkMsg(msg) {
```

Comment:
I agree that we should keep PRs small and address only a few things in each of them, but this is the actual PR that is introducing this code, and I agree with @zelig, that this should be changed to `validate() error`, so actually here is where we should be addressing this issue.